src/PendingObjects/TestCall.php

```diff
@@ -61,9 +61,13 @@ public function __construct(TestSuite $testSuite, string $filename, string $desc
      */
     public function throws(string $exceptionClass, string $exceptionMessage = null): TestCall
```

Comment:
@lukeraymonddowning Can you rename `$exceptionClass` to just `$exception`?

Reply:
Updated